Mathlib/Combinatorics/SimpleGraph/Finsubgraph.lean

```diff
@@ -49,6 +49,8 @@ abbrev FinsubgraphHom (G' : G.Finsubgraph) (F : SimpleGraph W) :=
 
 local infixl:50 " →fg " => FinsubgraphHom
 
+namespace Finsubgraph
```

Comment:
I think all these changes can go in a separate PR. Just
```
git checkout master
git checkout -b fancy-new-branch-name
git checkout frame_heyting_algebra -- Mathlib/Combinatorics/SimpleGraph/Finsubgraph.lean
git commit
gh pr create
```

Reply:
Thanks for the tip! Maybe I'm still a git nood after all...

#15005

Reply:
Those git commands look like a bad idea to me, that reverts all intermediate changes to `Finsubgraph`.